resources/public/api/conf/1.0/sandbox/working-tax-credits-response.md

```diff
@@ -48,11 +48,11 @@
     <tr>
           <td>fromDate requested is earlier than available data</td>
           <td>
-            <p>fromDate earlier than 2015.</p> 
+            <p>fromDate earlier than 6 April 2015.</p> 
             <p>For example, fromDate=2015-02-28</p>
           </td>
           <td><p>400 (Bad Request)</p>
-          <p>{ &quot;code&quot; : &quot;INVALID_REQUEST&quot;,<br/>&quot;message&quot; : &quot;startDate earlier than 31 March 2015" }</p></td>
+          <p>{ &quot;code&quot; : &quot;INVALID_REQUEST&quot;,<br/>&quot;message&quot; : &quot;fromDate earlier than 31 March 2015" }</p></td>
     </tr>
     <tr>
           <td>Invalid date format</td>
```

Comment:
"The font looks bigger for this row. Update so that it matches the font size in the rest of the table."

Reply:
See also incorrect matchId - I don't know if it works yet, but I addressed this in matching by wrapping the text in `<p></p>` so that everything's consistent